app/assets/javascripts/autocompletes.js

```diff
@@ -1,105 +1,113 @@
-	if($("#nav-global-search").has($(e.target)).length) {
-		e.preventDefault();
-		$("#nav-global-search").removeClass('minimized').find("input.search").focus();
-	} else {
-		$("#nav-global-search").addClass("minimized");
-	}
+  if ($("#nav-global-search").has($(e.target)).length) {
+    e.preventDefault();
+    $("#nav-global-search").removeClass('minimized').find("input.search").focus();
+  } else {
+    $("#nav-global-search").addClass("minimized");
+  }
 })
 
 // Initialize the global search on each page/turbolinks load.
 $(document).on('turbolinks:load', function() {
 
-	var allRecords = new Bloodhound({
-	  datumTokenizer: Bloodhound.tokenizers.obj.whitespace(['name', 'email']),
-	  queryTokenizer: Bloodhound.tokenizers.whitespace,
-	  remote: {
-	    url: '/search.json?q=%QUERY',
-	    wildcard: '%QUERY'
-	  }
-	});
-	
-	$('.global-search').typeahead({
-		hint: true,
-		highlight: true,
-		minLength: 1
-	},
-	{
-		name: 'all-records',
-		display: 'fullname',
-		source: allRecords,
-		limit: 10,
-		templates: {
-	    empty: "<p class='text-warning'>Sorry, couldn't find anything!</p>",
-	    suggestion: function(data) { return suggestionContent(data); },
-			pending: "<p class='loading'><i class='fa fa-spin fa-spinner'></i> Searching...</p>",
-			footer: "<p class='small text-muted footer'><b>Tip:</b> Search at any time by pressing the <kbd>/</kbd> key.</p>"
-	  }
-	})
-	.attr("spellcheck", "false")
-	.attr("autocomplete", "off")
-	.attr("autocapitalize", "off")
-	.on('typeahead:select', function(ev, suggestion) {
-		if($(this).data('action') == 'navigate') {
-		  Turbolinks.visit(suggestion.url);
-		} else {
-			alert("This feature is still under construction.")
-		}
-	});
-	
-	// Focus on the global search if the "/" key is pressed.
-	$(document).keydown(function(event) {
-		if ( $(event.target).is('input, textarea, select, [contenteditable]') ) { return; }
-		if ( event.which == 191 ) {
-	   event.preventDefault();
-		 $("#nav-global-search").removeClass('minimized').find("input.search").focus();
-	  }
-	})
-	
-})
+    var allRecords = new Bloodhound({
+      datumTokenizer: Bloodhound.tokenizers.obj.whitespace(['name', 'email']),
+      queryTokenizer: Bloodhound.tokenizers.whitespace,
+      remote: {
+        url: '/search.json?q=%QUERY',
+        wildcard: '%QUERY'
+      }
+    });
+
+    $('.global-search').typeahead({
+        hint: true,
+        highlight: true,
+        minLength: 1
+      }, {
+        name: 'all-records',
+        display: 'fullname',
+        source: allRecords,
+        limit: 10,
+        templates: {
+          empty: "<p class='text-warning'>Sorry, couldn't find anything!</p>",
+          suggestion: function(data) {
+            return suggestionContent(data);
+          },
+          pending: "<p class='loading'><i class='fa fa-spin fa-spinner'></i> Searching...</p>",
+          footer: function(query, suggestions) {
+            return "<a href='/search?q=" + query.query + "'>Advanced Search</a>" +
```

Comment:
I don't know much about the typeahead plugin being used here, but this looks like DOM based XSS waiting to happen